src/android/notification/util/AssetUtil.java

```diff
@@ -94,7 +95,70 @@ public Uri parse (String path) {
             return Uri.parse(path);
         }
 
-        return Uri.EMPTY;
+        return resolveFile(path);
```

Comment:
Minor, could be instead

```suggestion
        return resolveAssetFromFileName(path);
```

Unless you feel that's too verbose. Alternatively `resolveAssetFile`, just to make it clearer that it's via `AssetManager`